pom.xml

```diff
@@ -343,6 +343,11 @@
                 <artifactId>junit</artifactId>
                 <version>4.13.1</version>
             </dependency>
+            <dependency>
+                <groupId>org.testng</groupId>
+                <artifactId>testng</artifactId>
+                <version>7.4.0</version>
```

Comment:
Shouldn't we have test dependencies in the `<scope>test</scope>`? This also applies to the junit dependency

Reply:
Right, I'll look into this.